src/main/java/core/basesyntax/service/impl/ParseServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ParseService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParseServiceImpl implements ParseService {
+    private static final int ZERO_INDEX = 0;
+    private static final int FIRST_INDEX = 1;
+    private static final int SECOND_INDEX = 2;
+    private static final String COMA = ",";
+
+    @Override
+    public List<Transaction> transactionsParser(List<String> transactions) {
+        transactions.remove(ZERO_INDEX);
+        return transactions.stream()
+                .map(this::getTransaction)
+                .collect(Collectors.toList());
+    }
+
+    private Transaction getTransaction(String string) {
+        String[] splittedStrings = string.split(COMA);
+        return new Transaction(splittedStrings[ZERO_INDEX].trim(),
+                new Fruit(splittedStrings[FIRST_INDEX].trim()),
+                Integer.parseInt(splittedStrings[SECOND_INDEX].trim()));
```

Comment:
csv files usually do not contain redundant whitespaces

Reply:
I don't need to use it at the end of every splitted string, I should use it before split() method of String to delete System.line.separator.